tailwind.config.js

```diff
@@ -1,11 +1,44 @@
 module.exports = {
-  purge: [],
+  purge: ["src/**/*.{ts,tsx}"],
   darkMode: false, // or 'media' or 'class'
   theme: {
-    extend: {},
+    extend: {
+      colors: {
+        violet: {
+          light: "#C4D1FE",
+          dark: "#3C3CBE",
+          DEFAULT: "#6465F1",
+        },
+      },
+    },
+    container: {
+      padding: {
```

Comment:
Are you changing this because the default container padding is not what you want?

Reply:
No. I didn't know there are default properties. I just tried to start working with the prepared setup.